ui/src/album/AlbumListView.js

```diff
@@ -120,6 +120,7 @@ const AlbumListView = ({
               <br />
               <RatingField
                 record={r}
+                sortByOrder={'DESC'}
```

Comment:
Here it is not necessary, it is only used for making the header clickable in the DataGrid, not in SimpleLists